### PR TITLE
Insert text on double tap

### PR DESCRIPTION
**Pull request by Fausto95**
Fixes: https://github.com/excalidraw/excalidraw/pull/1179

 ### Test Plan

Tap twice to insert text.
Try zooming and make sure there's no text insertion

### CODE REVIEW

**Comment by dwelle** [comment]
Works great on android!

File: src/components/App.tsx [hunk 1]
```diff
@@ -403,6 +405,7 @@ export class App extends React.Component<any, AppState> {
     window.addEventListener("blur", this.onBlur, false);
     window.addEventListener("dragover", this.disableEvent, false);
     window.addEventListener("drop", this.disableEvent, false);
+    this.getCanvasElement()?.addEventListener("touchstart", this.onTapStart);
```
Comment (dwelle) [inline src/components/App.tsx:408]: Let's do this inside `<canvas>` `ref` handler. This will also remove the need for `getCanvasElement`

Comment (Fausto95) [reply src/components/App.tsx]: Oh yea. Just added it there

**Comment by j-f1** [comment]
Could this be expanded to watch `onPointerDown` instead so we don’t have to have the `onDblClick` handler?

**Comment by dwelle** [comment]
@j-f1 I suggested as much on chat. We can investigate if we can make this PR use `onPointerDown`, and as you say, remove `onDoubleClick` as well.

**Comment by Fausto95** [comment]
I was just about to investigate on that

**Pull request merged**